src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task{
+    protected String start;
+    protected String end;
```

Comment:
can consider more descriptive names instead of just start and end, maybe startOfEvent